scene/2d/physics_body_2d.cpp

```diff
@@ -1066,27 +1067,39 @@ Vector2 KinematicBody2D::move_and_slide(const Vector2 &p_linear_velocity, const
 	Vector2 body_velocity = p_linear_velocity;
 	Vector2 body_velocity_normal = body_velocity.normalized();
 	Vector2 up_direction = p_up_direction.normalized();
+	// Hack in order to work with calling from _process as well as from _physics_process; calling from thread is risky
```

Comment:
Nitpicking, but it would be nice to have a blank line before the comment to make things easier to read :)